### PR TITLE
[Bugfix] Triton pressureCurve

### PR DESCRIPTION
**Pull request by PhineasFreak**
**Change Log:**

* Fix the Triton pressure values (were high by a factor of 10) (issue https://github.com/KSP-RO/RealSolarSystem/issues/118).

### CODE REVIEW

**Review by eggrobin — APPROVED** [review]

File: GameData/RealSolarSystem/RSSKopernicus/Neptune/Triton.cfg [hunk 1]
```diff
@@ -157,23 +157,23 @@
...
-				key = 25000 0.00281212 -1.94787E-07 -1.94787E-07
-				key = 30000 0.00199469 -1.35817E-07 -1.35817E-07
-				key = 40000 0.00102311 -6.68765E-08 -6.68765E-08
-				key = 50000 0.000540320 -3.36465E-08 -3.36465E-08
-				key = 60000 0.000294696 -1.73705E-08 -1.73705E-08
-				key = 75000 0.000126524 -6.80449E-09 -6.80449E-09
-				key = 90000 5.86825E-05 -2.85739E-09 -2.85739E-09
-				key = 103000 3.20278E-05 -1.42561E-09 -1.42561E-09
+				key = 0 0.00165 0 -1.11539E-06
+				key = 2000 0.00143845 -1.00036E-06 -1.00036E-06
```
Comment (eggrobin) [inline GameData/RealSolarSystem/RSSKopernicus/Neptune/Triton.cfg]: The derivatives (third and fourth columns) need to be divided by ten too.

**Review by eggrobin — CHANGES_REQUESTED** [review]

**Review by eggrobin — APPROVED** [review]
This looks good to me (thanks for aligning the columns by the way, it's a whole lot more readable this way).

@Bornholio, can you test that the pressure behaves nicely now as you go from 0 m to 110 km?

**Comment by PhineasFreak** [comment]
Yea...whoops! I forgot that i had to change all pressure-related values.

Semi-offtopic but someone also reported that the Mars pressure curve may also be wrong/off. Will take a look at it.

**Comment by Bornholio** [comment]
starting in 20 minutes, also i noticed there is a huge variation on scientifically reported values for triton.

**Review by Bornholio — APPROVED** [review]
Looks Good, tested using barometer and other output values looks properly smoothed

**Pull request merged**